src/lib/ngsiNotify/Notifier.h

```diff
@@ -25,33 +25,37 @@
 *
 * Author: Fermin Galan
 */
-
 #include <map>
 #include <pthread.h>
 
+#include "common/NotificationFormat.h"
 #include "ngsi9/NotifyContextAvailabilityRequest.h"
 #include "ngsi10/NotifyContextRequest.h"
+#include "ngsiNotify/ThreadData.h"
 
-#include "ThreadData.h"
 
-class Notifier {
 
+/* ****************************************************************************
+*
+* Notifier -
+*/
+class Notifier
+{
 public:
-   
-    virtual ~Notifier(void);
-
-    virtual void sendNotifyContextRequest(NotifyContextRequest* ncr,
-                                          const std::string&    url,
-                                          const std::string&    tenant,
-                                          const std::string&    xauthToken,
-                                          const std::string&    fiwareCorrelator,
-                                          Format                format   = JSON);
-
-    virtual void sendNotifyContextAvailabilityRequest(NotifyContextAvailabilityRequest* ncr,
-                                                      const std::string&                url,
-                                                      const std::string&                tenant,
-                                                      const std::string&                fiwareCorrelator,
-                                                      Format                            format           = JSON);
+  virtual ~Notifier(void);
+
+  virtual void sendNotifyContextRequest(NotifyContextRequest* ncr,
+                                        const std::string&    url,
+                                        const std::string&    tenant,
+                                        const std::string&    xauthToken,
+                                        const std::string&    fiwareCorrelator,
+                                        NotificationFormat    notifyFormat = NGSI_V2_NORMALIZED);
```

Comment:
And, if we decided to vote on whether to forbid default values in the broker, if would definitely vote in favor.